src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -581,27 +620,33 @@ blockToMarkdown' opts t@(Table _ blkCapt specs thead tbody tfoot) = do
   return $ nst (tbl $$ caption'') $$ blankline
 blockToMarkdown' opts (BulletList items) = do
   contents <- inList $ mapM (bulletListItemToMarkdown opts) items
-  return $ (if isTightList items then vcat else vsep) contents <> blankline
+  return $ (if isTightList items then vcat else vsep) 
+                  contents <> blankline
 blockToMarkdown' opts (OrderedList (start,sty,delim) items) = do
   variant <- asks envVariant
   let start' = if variant == Commonmark || isEnabled Ext_startnum opts
                   then start
                   else 1
   let sty'   = if isEnabled Ext_fancy_lists opts then sty else DefaultStyle
-  let delim' = if isEnabled Ext_fancy_lists opts
-                  then delim
-                  else if variant == Commonmark &&
-                           (delim == OneParen || delim == TwoParens)
-                       then OneParen -- commonmark only supports one paren
-                       else DefaultDelim
+  let delim' | isEnabled Ext_fancy_lists opts =
+               case variant of 
+                   -- Markua supports 'fancy' enumerators, but no TwoParens
+                   Markua -> if delim == TwoParens then OneParen else delim
+                   _ -> delim
+             | variant == Commonmark && --commonmark only supports one paren
+                   (delim == OneParen || delim == TwoParens) = OneParen
+             | otherwise = DefaultDelim
   let attribs = (start', sty', delim')
   let markers  = orderedListMarkers attribs
-  let markers' = map (\m -> if T.length m < 3
-                               then m <> T.replicate (3 - T.length m) " "
-                               else m) markers
+  let markers' = case variant of
+                        Markua -> markers
+                        _ -> map (\m -> if T.length m < 3
+                                   then m <> T.replicate (3 - T.length m) " "
+                                   else m) markers
   contents <- inList $
               zipWithM (orderedListItemToMarkdown opts) markers' items
-  return $ (if isTightList items then vcat else vsep) contents <> blankline
+  return $ (if isTightList items then vcat else vsep) 
+                contents <> blankline
```

Comment:
Let's restore the original (lack of) indentation if it fits in 80 columns.

Reply:
its back to one line